app/helpers/application_helper.rb

```diff
@@ -39,12 +39,23 @@ def induction_coordinator_mentor_path(user)
     induction_coordinator_dashboard_path(user)
   end
 
+  def latest_manageable_cohort(school)
```

Comment:
I see this method is only used in controllers, so I perhaps it should be a service class and add some specs?

Reply:
Yes, you are right. I just placed it there to speed up bug fixes. 
I have now extracted it on to its own service.
Thanks @leoapost